README.md

```diff
@@ -95,10 +88,6 @@ See also [*delaunay*.renderTriangle](#delaunay_renderTriangle).
 
 The incoming halfedge indexes as a Int32Array [*e0*, *e1*, *e2*, …]. For each point *i*, *inedges*[*i*] is the halfedge index *e* of an incoming halfedge. For coincident points, the halfedge index is -1; for points on the convex hull, the incoming halfedge is on the convex hull; for other points, the choice of incoming halfedge is arbitrary. The *inedges* table can be used to traverse the Delaunay triangulation; see also [*delaunay*.neighbors](#delaunay_neighbors).
 
-<a href="#delaunay_outedges" name="delaunay_outedges">#</a> <i>delaunay</i>.<b>outedges</b>
-
-The outgoing halfedge indexes as a Int32Array [*e0*, *e1*, *e2*, …]. For each point *i* on the convex hull, *outedges*[*i*] is the halfedge index *e* of the corresponding outgoing halfedge; for other points, the halfedge index is -1. The *outedges* table can be used to traverse the Delaunay triangulation; see also [*delaunay*.neighbors](#delaunay_neighbors).
-
```

Comment:
Let's document `hullIndex` instead maybe?

Reply:
I don't think it should be in the public API:
- we have access to the hull if they need to reconstruct it, it's a linear op in time and memory
- we should use neighbors(i) anyway

(Maybe it should be private?)

Reply:
I don't know how to do that (ie: keeping this.hullIndex private) in a proper way.

Reply:
Ah don't worry about it, let's just leave it there without docs.

Reply:
I suggest either adding documentation for hullIndex to the README, or renaming it to _hullIndex to mark it clear that it’s private.